docs/turborepo.md

```diff
@@ -0,0 +1,36 @@
+# Turborepo and Telescope
+
+Telescope is configured as a monorepo: a single repository that holds many projects. [Turborepo](https://turborepo.org/) is the tool we have chosen to manage every project within Telescope.
+
+You will probably not have to interact with Turborepo directly but it is worth understanding how it work. The [Turborepo docs](https://turborepo.org/docs) do a good job at explaining its benefits and features.
```

Comment:
Typo: "it is worth understanding how it work" should be "it is worth understanding how it *works*".